provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/BulkEnrollmentOperationError.java

```diff
@@ -37,53 +38,23 @@
     private static final String REGISTRATION_ID_TAG = "registrationId";
     @Expose
     @SerializedName(REGISTRATION_ID_TAG)
+    @Getter
     private String registrationId;
 
     // the error code
     private static final String ERROR_CODE_TAG = "errorCode";
     @Expose
     @SerializedName(ERROR_CODE_TAG)
+    @Getter
     private Integer errorCode;
 
     // the error status
     private static final String ERROR_STATUS_TAG = "errorStatus";
     @Expose
     @SerializedName(ERROR_STATUS_TAG)
+    @Getter
```

Comment:
```suggestion
    /**
     * Getter for the error status.
     *
     * @return The {@code String} with the errorStatus content. It can be {@code null}.
     */
     @Getter
```